mypy/plugin.py

```diff
@@ -302,13 +301,23 @@ def get_method_hook(self, fullname: str
 
     def get_class_decorator_hook(self, fullname: str
                                  ) -> Optional[Callable[[ClassDefContext], None]]:
-        if fullname in mypy.plugins.attrs.attr_class_makers:
-            return mypy.plugins.attrs.attr_class_maker_callback
-        elif fullname in mypy.plugins.attrs.attr_dataclass_makers:
+        from mypy.plugins import attrs
+        from mypy.plugins import dataclasses
+
+        if fullname in attrs.attr_class_makers:
+            return attrs.attr_class_maker_callback
+        elif fullname in attrs.attr_dataclass_makers:
             return partial(
-                mypy.plugins.attrs.attr_class_maker_callback,
+                attrs.attr_class_maker_callback,
                 auto_attribs_default=True
             )
+        # TODO: It's unclear if fullname should be a fully qualified
```

Comment:
I believe this happens if the module is not in the typeshed.

Reply:
Thanks! Should I add the module to the typeshed first?

Reply:
I am fine with this solution. We can check later if this is fixed when the stub is in typeshed.

Reply:
There is already an open typeshed PR to add a dataclasses stub: https://github.com/python/typeshed/pull/1944. I found some issues in it, but the original author hasn't responded for a while. @Bogdanp if you're interested, you could take over that PR and I can help you get it merged into typeshed.